droolsjbpm-uber-distribution/src/main/assembly/assembly-droolsjbpm-uber.xml

```diff
@@ -22,7 +22,8 @@
       <includes>
         <include>org.drools:drools-distribution:zip</include>
         <include>org.drools:droolsjbpm-integration-distribution:zip</include>
-        <include>org.drools:droolsjbpm-tools-distribution:zip</include>
+        <!-- disable dependency to droolsjbpm-tools-distribution:zip since droolsjbp-tools was temporary disabled from repository-list -->
```

Comment:
I would say it was disable permanently. No one is going to add this back in the future :)